src/core.js

```diff
@@ -247,6 +258,13 @@ function date(format) {
           result += minutes;
           break;
 
+        // Seconds
+        case 's':
+          var seconds = d.getSeconds();
+          if (seconds.length == 1) { seconds = '0' + seconds; }
+          result += seconds;
+          break;
+
```

Comment:
Splendid.  Since I'm a new user of pull requests on Github, I don't know if your positive review is meant to apply only to the lines I see, 264-267, or to the whole patch.  Is there anything I need to do?  Do you want me to submit another patch without the backslash support?